src/components/Modals/FiatRamps/fiatRampProviders/onramper/index.ts

```diff
@@ -56,8 +56,10 @@ const generateSignature = async (data: string): Promise<string> => {
 export const createOnRamperUrl = async ({
   action,
   assetId,
+  amountCryptoPrecision,
   address,
   fiatCurrency,
+  fiatAmount,
```

Comment:
Confirmed gucci re: existing fiat ramps , tracked all the way down to existing callsites:

<img width="1279" height="312" alt="Image" src="https://github.com/user-attachments/assets/a2ea2ae8-634d-44d2-9770-629d5646e1b9" />

See `amountCryptoPrecision` / `fiatAmount` missing on old fiat amounts (right)